Jenkinsfile

```diff
@@ -39,128 +40,158 @@ node (node_label) {
 		load "pipeline-scripts/scale_ci_watcher.groovy"
 	}
 
+        if (pipeline == "TRUE") {
+		env.PIPELINE_STAGE=1
+		if ( build_tracker == "TRUE") {
+			load "pipeline-scripts/scale_ci_build_tracker.groovy"
+		}
+		if (openshiftv4_install_on_aws == "TRUE") {
+			load "pipeline-scripts/openshiftv4_on_aws.groovy"
+		}
+		if (openshiftv4_install_on_azure == "TRUE") {
+			load "pipeline-scripts/openshiftv4_on_azure.groovy"
+		}
+		if (http == "TRUE") {
+			load "pipeline-scripts/http.groovy"
+		}
+		if (run_uperf == "TRUE") {
+			load "pipeline-scripts/uperf.groovy"
+		}
+		if (ocpv4_scale == "TRUE") {
+                	load "pipeline-scripts/openshiftv4_scale.groovy"
+		}
+
+		env.PIPELINE_STAGE=2
+		if (http == "TRUE") {
+			 load "pipeline-scripts/http.groovy"
+		}
+		if (ocpv4_scale == "TRUE") {
+			load "pipeline-scripts/openshiftv4_scale.groovy"
+		}
+	}
+
 	// Queries UMB message to capture the OCP 4.x payloads
-	if ( build_tracker == "TRUE") {
+	if ( build_tracker == "TRUE" && pipeline != "TRUE") {
```

Comment:
Minor nit but this should probably be an if/else type statement instead of if pipeline != true for each and every if after the pipeline if true. Would make it more readable and all. For example,
```
if (pipeline == "TRUE") {
   do stuff
else
  do the stuff w/o the pipeline
}
```

Reply:
makes sense, thanks russ
quick test after the change https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/scale-ci-pipeline-test/259/console